CorsixTH/Lua/world.lua

```diff
@@ -649,6 +649,7 @@ function World:tickEarthquake()
       self.next_earthquake.damage_timer = self.next_earthquake.damage_timer + earthquake_damage_time
     end
 
+    --[[
```

Comment:
Just remove dead code, don't comment it out (source control is for recovering if desired)